src/api/app/controllers/webui/request_controller.rb

```diff
@@ -228,18 +222,13 @@ def add_role_request
     req = nil
     begin
       BsRequest.transaction do
-        req = BsRequest.new
-        req.state = 'new'
-        req.description = params[:description]
-
         opts = { target_project: params[:project],
-                 role:           params[:role] }
+                 role:           params[:role],
+                 description: params[:description] }
         opts[:target_package] = params[:package] if params[:package]
         opts[:person_name] = params[:user] if params[:user]
         opts[:group_name] = params[:group] if params[:group]
-        action = BsRequestActionAddRole.new(opts)
-        req.bs_request_actions << action
-
+        req = prepare_bsrequest(:add_role, opts)
```

Comment:
what about passing `params[:description]` instead of adding it to opts and removing it after using it inside the method :see_no_evil:

Reply:
you mean the method get a third parameter ```description```?

Reply:
I see no adventages of tons of parameters

Reply:
You could use params.slice(:description, ,,,), but there is to much mismatch between the key names in opts and params

Reply:
params is not an array, but a PermittedParams object - slicing that is not allowed

Reply:
@coolo what do you mean? do don't like to add more things to the `param` hash or giving more params to the method?

Reply:
my answer to your initial 'what about' question is: no, thanks